.github/workflows/testing.yml

```diff
@@ -370,31 +370,6 @@ jobs:
           test/test_cvedb.py
       - name: Run HTML tests
         run: pytest -v -n auto test/test_html.py
-      - name: Cache conda
-        uses: actions/cache@v3
-        env:
-          # Increase to reset cache if requirements.txt file has not changed
-          CACHE_NUMBER: 0
-        with:
-          path: ~/conda_pkgs_dir
-          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
-            hashFiles('requirements.txt') }}
-      - uses: conda-incubator/setup-miniconda@v2
-        with:
-          auto-update-conda: true
-          activate-environment: pdftotext
-          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
-      - name: Install pdftotext, reportlab and cve-bin-tool
```

Comment:
I didn't know if I was supposed to delete this section or only the pdftotext lines. Feel free to correct me :)